test-packages/integration-tests/test/v2/filter-functions.spec.ts

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+import {
```

Comment:
genral comment: In the `filter-factory.ts` of the test util I see many filter object beeing created for testing. But this has not been adjusted for all the new filter function beeing defined here. Is this obvious?

Reply:
As discussed: Only do the specific cases.
Add type tests to `filter.spec.ts`.

Reply:
Added some more tests as discussed.